lib/web_ui/lib/src/engine/platform_dispatcher/view_focus_binding.dart

```diff
@@ -39,9 +39,17 @@ final class ViewFocusBinding {
     }
   }
 
+  late final DomEventListener _handleFocusin = createDomEventListener(
+    (DomEvent event) => _handleFocusChange(event.target as DomElement?),
+  );
+
+  late final DomEventListener _handleFocusout = createDomEventListener(
+    (DomEvent event) => _handleFocusChange(event.relatedTarget as DomElement?),
```

Comment:
`relatedTarget` should be added to a `DomFocusEvent` class, it doesn't exist in the root `DomEvent` class. See:

* https://developer.mozilla.org/en-US/docs/Web/API/Event/Comparison_of_Event_Targets
* https://developer.mozilla.org/en-US/docs/Web/API/FocusEvent#instance_properties

Reply:
Done